lib/wrappers/openssl.nim

```diff
@@ -26,7 +26,31 @@
 
 const useWinVersion = defined(Windows) or defined(nimdoc)
 
-when useWinVersion:
+## To force openSSL version use -d:sslVersion=1.0.0
+## See: #10281, #10230
+## General issue:
+## Other dynamic libraries (like libpg) load diffetent openSSL version then what nim loads.
+## Having two different openSSL loaded version causes a crash.
+## Use this compile time define to force the openSSL version that your other dynamic libraries want.
```

Comment:
Note that the doc gen will show this as one big paragraph which likely isn't your intention.

Reply:
You right. I will change it to a single comment so that it does not show up in docs.

Reply:
Well, it would be nice to have this somewhere in our docs :)